### PR TITLE
Uprev the base image from trusty to bionic

### PR DESCRIPTION
**Pull request by SimonKrughoff**
This image only makes Python 3.4.3 available, but some of the new documentation tooling uses f-strings.  These require Python 3.7 to function.  I'm suggesting just upping the base image.  I have not tested whether package names have changed in  the interim, but I hope they have not.

### CODE REVIEW

**Comment by SimonKrughoff** [comment]
OK.  So travis failed not being able to find a package name.  This will require a little more looking.

**Comment by SimonKrughoff** [comment]
It seems like the only offending package is `latex-xcolor`.

**Comment by SimonKrughoff** [comment]
Travis has passed.  I think this is now ok to merge, but I don't know if I should do more complete testing.

**Review by jonathansick — CHANGES_REQUESTED** [review]
Please squash the last two commits.

Once this is merged to master it’ll build a new `latest` docker image from Travis and get used from lsst-texmf.

Note that in order for the change to be useful you’ll need to install third party dependencies required by db2author.py in the lsst-texmf image.

**Comment by SimonKrughoff** [comment]
Rebased.  I am doing the changes to `lsst-texmf` on a different PR [here](https://github.com/lsst/lsst-texmf/pull/196).

**Review by jonathansick — APPROVED** [review]

**Pull request merged**
